python/modem-firmware-1.3+/nrf_cloud_diap.py

```diff
@@ -96,7 +96,7 @@ def create_provisioning_cmd_finished(api_key, dev_uuid, description='Provisionin
     return requests.post(req, json=payload, headers=get_auth_header(api_key))
 
 def create_provisioning_cmd_csr(api_key, dev_uuid, description='Generate CSR',
-                                attributes='O=Nordic Semiconductor,L=Oslo,C=fi',
+                                attributes='',
```

Comment:
why removed?

Reply:
it was only the default value since that what was called for in the diap test plan.
it is a now a parameter so users can add the appropriate values.